iphone/Classes/GestureModule.h

```diff
@@ -1,26 +1,30 @@
 /**
  * Appcelerator Titanium Mobile
- * Copyright (c) 2009-2010 by Appcelerator, Inc. All Rights Reserved.
+ * Copyright (c) 2009-Present by Appcelerator, Inc. All Rights Reserved.
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
-#import <TitaniumKit/TiModule.h>
-
 #ifdef USE_TI_GESTURE
+#import <JavaScriptCore/JavaScriptCore.h>
+#import <TitaniumKit/ObjcProxy.h>
 
-@interface GestureModule : TiModule {
-  NSTimeInterval lastShakeTime;
-}
+@protocol GestureExports <JSExport>
+
+// Properties (and accessors)
+READONLY_PROPERTY(BOOL, landscape, Landscape);
+READONLY_PROPERTY(NSNumber *, orientation, Orientation);
+READONLY_PROPERTY(BOOL, portrait, Portrait);
 
-@property (nonatomic, readonly) NSNumber *orientation;
+// methods
+// TODO: Add isFaceDown() and isFaceUp()?
+// - (BOOL)isFaceDown;
+// - (BOOL)isFaceUp;
```

Comment:
I think these got removed a while back? I only see them listed for Windows now in our docs, and that may be only because we didn't remember to remove it there...